docs/appconfig/schema-definitions-dependencyendpoint-properties-apipath.md

```diff
@@ -1,10 +1,10 @@
-# Untitled undefined type in AppConfig Schema
+# Untitled string in AppConfig Schema
 
 ```txt
 https://cloud.redhat.com/schemas/clowder-appconfig#/definitions/DependencyEndpoint/properties/apiPath
 ```
 
-The top level api path that the app should serve from /api/<apiPath>
+The top level api path that the app should serve from /api/<apiPath> (deprecated, use apiPaths)
```

Comment:
I guess the api docs did not get updated in the prior apiPaths MR's ... anything related to 'apiPaths' changing in the docs here is due to that.

Reply:
#888 will fix this.